features/atomic/atomic_mount_unmount.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Atomic mount and unmount test for new upgrade tree
+    Describes tests for mount/umount to new ostree
+
+Background: Atomic hosts are discovered
+      Given "cihosts" hosts from dynamic inventory
+
+  Scenario: 1. Host provisioned and subscribed
+      Given "cihosts" host
+       When "cihosts" host is auto-subscribed to "stage"
+       Then subscription status is ok on "cihosts"
+        and "1" entitlement is consumed on "cihosts"
+
+  Scenario: 2. 'atomic host upgrade' is successful
+      Given there is "1" atomic host tree deployed
+       When atomic host upgrade is successful
+       Then wait "60" seconds for "cihosts" to reboot
+       Then there is "2" atomic host tree deployed
+
+  Scenario: 3. docker run busybox with detach mode
+      Given "docker" is already installed on "cihosts"
+        and "docker" is already running on "cihosts"
+       When docker pull "busybox"
+        and docker run "busybox" detach mode with "/usr/bin/top -b"
+       Then check whether there is a running container
+
+  Scenario: 4. mount container image to a specified directory
+       When atomic mount container to a specified "/mnt"
+       Then check whether atomic mount point "/mnt" exists
+
+  Scenario: 5. unmount a container image previously mounted
+       When atomic unmount a container image from previous "/mnt"
+       Then check whether atomic mount point "/mnt" does not exist
+
+  Scenario: 6. Rollback to the original deployment
```

Comment:
Is there a need to perform a rollback before stopping the container and removing the image?